app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt

```diff
@@ -125,6 +126,7 @@ class NewCreatePinFragment :
         } else {
             arrayListOf()
         }
+        (activity as? RootActivity)?.onSignInSuccess()
```

Comment:
I missed that, thanks! 
you can remove the next line, because onSignInSuccess in activity does all necessary navigation stuff
```
 popAndReplaceFragment(
            MainContainerFragment.create(actions),
            inclusive = true
```

Reply:
ah I see, you've changed onSignInSuccess